blogposts/2022/release-3.38.md

```diff
@@ -0,0 +1,74 @@
+---
+title: 'Sourcegraph 3.38 release'
+publishDate: 2022-03-21T10:00-07:00
+description: 'Sourcegraph 3.38 introduces improved Notebooks, faster Code Intelligence for large repositories, dependencies search, and custom file syntax highlighting.'
+tags: [blog, release]
+slug: 'release/3.38'
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.38/sourcegraph-3-38-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.38/sourcegraph-3-38-release.png
+changelogItems:
+  - description: Code Insights will now periodically clean up data series that are not in use. There is a 1 hour grace period where the series can be reattached to a view, after which all of the time series data and metadata will be deleted. This means customers deploying their own instance won't be storing data that has no use.
+    url: https://github.com/sourcegraph/sourcegraph/pull/32094
+    category: Code Insights
+  - description: 'Notebooks from private enterprise instances can now be embedded in external sites by enabling the `enable-embed-route` feature flag. This allows you to embed Notebooks within existing technical documentation to see your documentation and code side-by-side.'
+    url: https://docs.sourcegraph.com/notebooks/notebook-embedding
+    category: Notebooks
+  - description: LSIF upload pages now include a section listing the reasons and related retention policies resulting in an upload being retained and not expired.
+    url: https://github.com/sourcegraph/sourcegraph/pull/30864
+    category: Code Intelligence
+  - description: 'Code monitoring now has a Logs tab enabled as a [Beta feature](https://docs.sourcegraph.com/admin/beta_and_experimental_features). This lets you see recent runs of your code monitors and determine if any notifications were sent or if there were any errors during the run.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/32292
+    category: Admin
+  - description: 'Searching for the pattern `//` with regular expression search is now interpreted literally and will search for `//`. Previously, the `//` pattern was interpreted as our regular expression syntax `/<regexp>/` which would in turn be intrpreted as the empty string. Since searching for an empty string offers little practical utility, we now instead interpret `//` to search for its literal meaning in regular expression search.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/31520
+    category: Search
+---
+
+Sourcegraph 3.38 is now available! For this release, we introduced:
```

Comment:
This isn't blocking feedback but something to consider for next time or whenever we have lots to share in a release post: consider having an intro paragraph that briefly summarizes all the release highlights. Otherwise there's a long way to scroll to see Dependencies search, for example. We might catch people with something they're interested in if we surface it earlier in the post 😃